### ISSUE TITLE
Text appearing in the top-left 

### ISSUE THREAD

**Issue by kinwolfqc** [mlavin/django-selectable#174]
I'm updating an app from django 1.9 to django 1.10 and I got a weird behavior with django-selectable and django 1.10.1.  In django 9, on the html page, when you type  the autocomplete box with the result will correctly appear under the stringfield, but in django 10 it will appear in the top left corner of the browser.  Bug?

**Comment by mlavin** [comment]
Is this behavior in your own form/front-end or in the admin?

**Comment by kinwolfqc** [comment]
Good point as this reminded me that I also started using django-crispy-forms in the updated version.  The problem happens in the front-end form, and that form is built using the crispy tag` {% crispy form form.helper %}`.   The form.helper is there to give custom `<div>` tags to the different form fields.

I'm not using django-selectable in the admin section.

**Comment by kinwolfqc** [comment]
Ok, found the problem.  Jquery 3.1.1 and JQuery UI 1.11.4 have problems together.  I was getting a "f.getClientRects is not a function".  Supposedly that this is fixed with jquery UI 1.12.x but dj-selectable doesn't seem to play nice with that one(give a zIndex error)  So  I had to add jquery-migrate 3.0 to the template fir jquery ui 1.11.4 to work correctly and it's working ok once more now.
